web/src/settings_org.ts

```diff
@@ -457,6 +457,21 @@ function update_test_welcome_bot_custom_message_button_status(): void {
     $test_message_button.prop("disabled", true);
 }
 
+export function resize_settings_textarea(textarea_id: string): void {
+    const $el = $<HTMLTextAreaElement>(`#${CSS.escape(textarea_id)}`);
+
+    const min_rows = 2;
+    const max_rows = 5;
+    $el.attr("rows", min_rows);
```

Comment:
What is the need to set `rows` attribute here when it is being set in the end?

Reply:
Surprisingly, it works for me after removing this line. Also theoretically, it should not be required because we anyways set the height at the end of this function and `scrollHeight` or `line_height` should not depend on the `rows` attribute.
![Peek 2025-09-18 12-50](https://github.com/user-attachments/assets/12998bab-20b9-4ea6-b0ee-e41aed319d46)